receiver/journaldreceiver/README.md

```diff
@@ -0,0 +1,26 @@
+## `Journald Receiver`
+
+Parses Journald events using using the [opentelemetry-log-collection](https://github.com/open-telemetry/opentelemetry-log-collection) library.
```

Comment:
Please rephrase as a complete sentence and remove the duplicate word.

Reply:
Done. Let me know if anything specific needs to be added. Thanks!